src/routes/safe/components/Transactions/TxList/TxSingularDetails.tsx

```diff
@@ -138,20 +53,22 @@ const TxSingularDetails = (): ReactElement => {
     )
   }
 
-  if (!indexedTx) {
+  const detailedTx = transaction || (fetchedTx ? makeTxFromDetails(fetchedTx) : null)
+
+  if (!detailedTx) {
     return (
       <Centered padding={10}>
         <Loader size="sm" />
       </Centered>
     )
   }
 
-  const { transaction, txLocation } = indexedTx
-  const TxList = isTxQueued(transaction.txStatus) ? QueueTxList : HistoryTxList
+  const isQueue = isTxQueued(detailedTx.txStatus)
```

Comment:
Why did you decide to revert the pending check?

Reply:
I thought you didn't like it.